packages/nextjs/.npmignore

```diff
@@ -1,4 +1,9 @@
+# Info: the paths in this file are specified so that they align with the file
+# structure in `./build` where this file is copied to. This is done by the
+# prepack script `sentry-javascript/scripts/prepack.ts`.
+
 *
+
 !/dist/**/*
 !/esm/**/*
-!/build/types/**/*
+!/types/**/*
```

Comment:
Question: Other than the temporary inclusion of `!/build/**/*` in the bundleful versions of this file, will its contents ever vary? If not, could we just have a top-level one and copy that into the right spot using the prepack script?

Reply:
UPDATE: Chatted IRL and I'm going to make these changes.